contributions/catalog/catalog_rest_service.py

```diff
@@ -114,6 +115,14 @@ def index():
         # query only published ones
         headers = requestutil.get_header_using_api_key()
         result = requestutil.request_contributions(headers)
+
+        if result.status_code != 200:
+            msg = {
+                "ERROR": "There is a problem getting contributions list. Maybe API Key is not correct."
+            }
+            msg_json = jsonutil.create_log_json("Contribution", "SEARCH", msg)
+            logging.info("Contribution SEARCH " + json.dumps(msg))
```

Comment:
This doesn't make any difference because msg will be overridden as the same as msg_json but I will change it to msg_json for clearer understanding of the code.